tests/test_helper/upgrade_procedures.bash

```diff
@@ -1,3 +1,87 @@
+# rejoin_cluster_with_tls CONTAINER DB_TYPE TARGET_IP TARGET_PROTO MONITOR1 [MONITOR2 ...]
+#
+# Rejoin OVN OVSDB cluster on with member running on CONTAINER using tls.
+#
+# This function requires DB_TYPE which must be either 'nb' or 'sb'. It then disconnects CONTAINER
+# from cluster and rejoin the cluster again using TARGET_IP and TARGET_PROTO (either 'tcp' or 'ssl').
+#
+# MONITORs should be list of remaining containers in the cluster that are used to verify that CONTAINER
+# left the cluster before attempting to rejoin it.
+function rejoin_cluster_with_tls() {
+    local target_container=$1; shift
+    local db_type=$1; shift
+    local target_ip=$1; shift
+    local target_proto=$1; shift
+    local monitor_containers=$*
+    local ctl_path="/var/snap/microovn/common/run/ovn/"
+    local db_path="/var/snap/microovn/common/data/central/db/"
+    local local_ip=""
+    local_ip=$(microovn_get_cluster_address "$target_container")
+    local target_server_id=""
+    target_server_id=$(microovn_ovndb_server_id "$target_container" "$db_type")
+
+    if [ "$db_type" == "nb" ]; then
```

Comment:
There is a function to do this:
https://github.com/canonical/microovn/blob/2a0e6b556a02074774dcc5fc50f835a8f2165b50/tests/test_helper/microovn.bash#L129-L135

It should probably be exported though, and not have a _ prefix